### PR TITLE
Update change log for Nuvoton Ethernet and Common I/O library.

### PR DESCRIPTION
**Pull request by qiutongs**
Description
-----------

Update change log for Nuvoton Ethernet and Common I/O library.

Checklist:
----------

- [ ] I have tested my changes. No regression in existing tests.
- [ ] My code is Linted.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,9 +4,10 @@
 
 ### New Features
 
-#### Common I/O Library V1.0.0
-- Common I/O library provides a set of standard APIs, across supported reference boards, for accessing common serial devices. 
-- Supported peripherals include UART, SPI and I2C.
+#### Common I/O Library V0.5.0
+- A preview of Common I/O library is included.
+- Common I/O library provides a set of standard APIs, across supported reference boards, for accessing common devices.
+- It will start with supporting serial peripherals for UART, SPI and I2C. Other peripheral support will follow.
```
Comment (dcgaws) [inline CHANGELOG.md]: "This preview supports UART, SPI and I2C. Support for other peripherals will follow."

**Comment by dcgaws** [comment]
Can you please change the capitalization of the description text to:

"Update change log for Nuvoton Ethernet and Common I/O library."

File: CHANGELOG.md (hunk 1 above)
Comment (qiutongs) [reply CHANGELOG.md]: > Can you please change the capitalization of the description text to:
> 
> "Update change log for Nuvoton Ethernet and Common I/O library."

Done.

**Comment by qiutongs** [comment]
> Can you please change the capitalization of the description text to:
> 
> "Update change log for Nuvoton Ethernet and Common I/O library."

Done.

File: CHANGELOG.md (hunk 1 above)
Comment (qiutongs) [reply CHANGELOG.md]: OK.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +18,10 @@
 #### OTA for NXP LPC54018 IoT Module
 - Over-the-air (OTA) updates are now available for LPC54018.
 
+#### Ethernet Support on Nuvoton NuMaker-IoT-M487
+- Ethernet porting code has been added for Nuvoton NuMaker-IoT-M487.
+- Either WI-FI or Ethernet can be enabled exclusively.
```
Comment (dcgaws) [inline CHANGELOG.md]: "Wi-Fi"

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +18,10 @@
 #### OTA for NXP LPC54018 IoT Module
 - Over-the-air (OTA) updates are now available for LPC54018.
 
+#### Ethernet Support on Nuvoton NuMaker-IoT-M487
+- Ethernet porting code has been added for Nuvoton NuMaker-IoT-M487.
```
Comment (dcgaws) [inline CHANGELOG.md]: "Ethernet is now available for Nuvoton NuMaker-IoT-M487."

File: CHANGELOG.md (hunk 2 above)
Comment (qiutongs) [reply CHANGELOG.md]: ok

File: CHANGELOG.md (hunk 3 above)
Comment (qiutongs) [reply CHANGELOG.md]: Sounds better.

**Pull request merged**